sections/04_study.md

```diff
@@ -478,114 +478,103 @@ within neural networks.
-spatial and temporal features to predict protein contacts[@doi:10.1093/bioinformatics/bts475].
-Eickholt and Cheng combined deep belief networks and boosting techniques to predict protein contacts
-[@doi:10.1093/bioinformatics/bts598] and trained deep networks by layer-wise unsupervised
-learning followed by fine-tuning of the entire network. Skwark and Elofsson et al. developed
-an iterative deep learning technique for contact prediction by stacking a series of Random Forests
-[@doi:10.1371/journal.pcbi.1003889]. However, blindly tested in the well-known CASP competitions,
-these methods did not show any advantage over MetaPSICOV [@doi:10.1093/bioinformatics/btu791], a method
-using two cascaded neural networks. Very recently, Wang and Xu et al. proposed a novel deep learning method
-RaptorX-Contact [@doi:10.1371/journal.pcbi.1005324] that can significantly improve contact prediction
-over MetaPSICOV especially for proteins without many sequence homologs. RaptorX-Contact employs a network
-architecture formed by one 1D residual neural network and one 2D residual neural network.
-Blindly tested in the latest CASP competition (i.e. CASP12 [@url:http://www.predictioncenter.org/casp12/rrc_avrg_results.cgi]),
-RaptorX-Contact is ranked first in terms of the total F1 score (a widely-used performance metric) on
-free-modeling targets as well as the whole set of targets. In the CASP12 test, the group ranked second
-also employed a deep learning method. Even MetaPSICOV, which ranked third in CASP12, employed more
-and wider hidden layers than its old version. Wang and Xu et al. have also
-demonstrated in another blind test CAMEO (which can be interpreted as a fully-automated
-CASP) [@url:http://www.cameo3d.org/] that their predicted contacts can help fold quite a few proteins
-with a novel fold and only 65-330 sequence homologs and that their method also works well on membrane
-protein contact prediction even if trained mostly by non-membrane proteins. In fact, most of the top 10
-contact prediction groups in CASP12 employed some kind of deep learning techniques. The RaptorX-Contact
-method performed better mainly due to introduction of residual neural networks and exploiting contact
-occurrence patterns by simultaneous prediction of all the contacts in a single protein.
-It is still possible to further improve contact prediction by studying new deep network architectures.
-However, current methods fail when proteins in question have almost no sequence homologs. It is unclear if there
-is an effective way to deal with this type of proteins or not except waiting for more sequence homologs.
-Finally, the deep learning methods summarized above also apply to interfacial contact prediction
-of a protein complex, but may be less effective since on average protein complexes have fewer sequence homologs.
+beta strand and loop regions) or eight finer-grained states. A predictor is
```

Comment:
Add comma after "beta strand".

The sentence "A predictor is..." could be clarified.  Maybe something like "Q3 and Q8 accuracy pertain to 3-state or 8-state predictions, respectively.